src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -106,9 +115,14 @@ def post(self, request, project):
             report.event_user_id = euser.id
 
         try:
-            report.group = Group.objects.from_event_id(project, report.event_id)
-        except Group.DoesNotExist:
-            pass
+            event = Event.objects.filter(event_id=report.event_id).select_related('group')[0]
```

Comment:
As-is, this query is unsafe. `event_id` is not globally unique and is only unique within a given `project`.

Reply:
(I also don't think we have an index on `event_id` that isn't qualified by the `project_id`.)